cli-init.sh

```diff
@@ -2,4 +2,4 @@
 cd packages
 
 # Initate template QA
-yo --no-insight --no-update-notifier ../cli/generator-op-microservice/generators/microservice
+../cli/generator-op-microservice/node_modules/yo/lib/cli.js --no-insight --no-update-notifier ../cli/generator-op-microservice/generators/microservice
```

Comment:
npm automatically creates a symlink in `node_modules/.bin` directory for all the executable npm commands.
```suggestion
../cli/generator-op-microservice/node_modules/.bin/yo --no-insight --no-update-notifier ../cli/generator-op-microservice/generators/microservice
```